### PR TITLE
New package: gprbuild-20180528

### PR DESCRIPTION
**Pull request by dachshund7**

### CODE REVIEW

File: srcpkgs/gprbuild/template [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	http://mirrors.cdn.adacore.com/art/5b0819dec7a447df26c27a40>xmlada-gpl-2018-20180524-src.tar.gz"
+checksum="a06b7ee6ddc00a927db9b6207a4bd44e670ff2cceb9879a5baa98f4a38f552a1
+	aacef3ae91e701690f60bde9c8a4a3f47ba8ca1f8f3c0599d01bce0173be80de"
+
+_bin_progs="gprbuild gprconfig gprclean gprinstall gprname gprls"
+_lib_progs="gprlib gprbind"
+
+do_build() {
+	cd gprbuild-gpl-2018-src
+	patch -Np0 -i ${FILESDIR}/gprbuild-2018.patch
```
Comment (Cogitri) [inline srcpkgs/gprbuild/template]: Why not put this in `./patches`?

File: srcpkgs/gprbuild/template [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+	srcdir=$PWD
+	xmlada_src=../xmlada-gpl-2018-src
+
+	CC=${CC:-cc}
```
Comment (Cogitri) [inline srcpkgs/gprbuild/template]: I don't think CC is ever not set

File: srcpkgs/gprbuild/template [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+	GNATMAKE=${GNATMAKE:-gnatmake}
+	CFLAGS=${CFLAGS:-$CFLAGS}
```
Comment (Cogitri) [inline srcpkgs/gprbuild/template]: Same for CFLAGS (and even so this doesn't make sense)

File: srcpkgs/gprbuild/template [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+	GNATMAKEFLAGS=${GNATMAKEFLAGS:--j0}
+
```
Comment (Cogitri) [inline srcpkgs/gprbuild/template]: Why not use XBPS_MAKEJOBS here?

File: srcpkgs/gprbuild/template [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+homepage="https://github.com/AdaCore/gprbuild"
+distfiles="http://mirrors.cdn.adacore.com/art/5b0819dfc7a447df26c27a68>gprbuild-gpl-2018-20180524-src.tar.gz
+	http://mirrors.cdn.adacore.com/art/5b0819dec7a447df26c27a40>xmlada-gpl-2018-20180524-src.tar.gz"
+checksum="a06b7ee6ddc00a927db9b6207a4bd44e670ff2cceb9879a5baa98f4a38f552a1
+	aacef3ae91e701690f60bde9c8a4a3f47ba8ca1f8f3c0599d01bce0173be80de"
+
+_bin_progs="gprbuild gprconfig gprclean gprinstall gprname gprls"
+_lib_progs="gprlib gprbind"
+
+do_build() {
```
Comment (Cogitri) [inline srcpkgs/gprbuild/template]: Can't you set wrksrc appropriately instead of cd-ing?

File: srcpkgs/gprbuild/template (hunk 4 above)
Comment (maxice8) [reply srcpkgs/gprbuild/template]: XBPS_MAKEJOBS is just a number, you want the `makejobs` variable

**Pull request closed without merging**